### PR TITLE
COG-1152 - Fix admin_name is not formatted as lastname, firstname

### PR DESCRIPTION
**Pull request by PradipKarki**
### JIRA Issue Link
- [COG-1152: admin_name and user_name in audit_event should be in "Last Name, First Name" format for all events](https://osi-cwds.atlassian.net/browse/COG-1152)

### Technical Description
Format admin_name and user_name as "Last Name, First Name" for all user events

### Tests
- [X] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

### Types of changes

- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Configuration changes
N/A

### Checklist:

- [X] My code is in a stable state ready to be deployable, but not necessarily complete.
- [X] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: src/main/java/gov/ca/cwds/util/CurrentAuthenticatedUserUtil.java [hunk 1]
```diff
@@ -26,8 +26,10 @@ public static String getCurrentUserCountyName() {
     return (String) getCurrentUser().getParameter(COUNTY_NAME_PARAM);
   }
 
-  public static String getCurrentUserFullName() {
-    return getCurrentUser().getParameter(FIRST_NAME) + " " + getCurrentUser().getParameter(LAST_NAME);
+  public static String formatUserFullName() {
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/util/CurrentAuthenticatedUserUtil.java]: having this method here in CurrentAuthenticatedUserUtil violates the single responsibility principle
consider refactoring 

you have a nice UserNameFormatter.formatUserFullName(User user) util method that can be used

File: src/main/java/gov/ca/cwds/idm/event/UserChangeLogEvent.java [hunk 2]
```diff
@@ -1,5 +1,7 @@
 package gov.ca.cwds.idm.event;
 
+import static gov.ca.cwds.util.UserNameFormatter.formatUserFullName;
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/event/UserChangeLogEvent.java]: is this needed?

Comment (sorych) [reply src/main/java/gov/ca/cwds/idm/event/UserChangeLogEvent.java]: see my comment above, please

File: src/main/java/gov/ca/cwds/util/CurrentAuthenticatedUserUtil.java (hunk 1 above)
Comment (PradipKarki) [reply src/main/java/gov/ca/cwds/util/CurrentAuthenticatedUserUtil.java]: Correct, I renamed back to getCurrentUserFullName() which uses internally formatUserFullName

File: src/main/java/gov/ca/cwds/idm/event/UserChangeLogEvent.java (hunk 2 above)
Comment (PradipKarki) [reply src/main/java/gov/ca/cwds/idm/event/UserChangeLogEvent.java]: Please review, I made changes.

**Pull request merged**